traitlets/config/tests/test_configurable.py

```diff
@@ -368,7 +368,7 @@ class SomeSingleton(SingletonConfigurable):
             pass
 
         class DefaultConfigurable(Configurable):
-            a = Integer(config=True)
+            a = Integer().tag(config=True)
             def _config_default(self):
                 if SomeSingleton.initialized():
                     return SomeSingleton.instance().config
```

Comment:
Sure. Which test(s) do you want?  I'll copy it and check for the deprecation warning too.